diff --git a/node_modules/@dr.pogodin/react-native-audio/android/src/main/java/com/drpogodin/reactnativeaudio/ReactNativeAudioPackage.kt b/node_modules/@dr.pogodin/react-native-audio/android/src/main/java/com/drpogodin/reactnativeaudio/ReactNativeAudioPackage.kt
index 175de52..ce91701 100644
--- a/node_modules/@dr.pogodin/react-native-audio/android/src/main/java/com/drpogodin/reactnativeaudio/ReactNativeAudioPackage.kt
+++ b/node_modules/@dr.pogodin/react-native-audio/android/src/main/java/com/drpogodin/reactnativeaudio/ReactNativeAudioPackage.kt
@@ -24,6 +24,7 @@ class ReactNativeAudioPackage : BaseReactPackage() {
         ReactNativeAudioModule.NAME,
         canOverrideExistingModule = false,  // canOverrideExistingModule
         needsEagerInit = false,  // needsEagerInit
+        hasConstants = false,
         isCxxModule = false,  // isCxxModule
         isTurboModule = true // isTurboModule
       )
